packages/block-editor/src/components/link-control/style.scss

```diff
@@ -168,35 +168,38 @@ $preview-image-height: 140px;
 		align-items: flex-start;
 		margin-right: $grid-unit-10;
 		overflow: hidden;
-
-		// Force text to wrap to improve UX when encountering long lines
-		// of text, particular those with no spaces.
-		// See: https://github.com/WordPress/gutenberg/issues/33586#issuecomment-888921188
```

Comment:
I'd be concerned about changing this. The referenced issue provides a clear rationale for why this was necessary.